### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by LDouglasOT**
**Tasks Completed**

- Add new column to the fields called species
245539063-a0cbcd5e-559d-4246-bd78-ba18c5f1c02a

- populate db with new objects to make 11 rows
svaes

- update animals , the species field.
Screenshot 2023-06-15 at 12 04 43

- Delete all the fields and rollback
Screenshot 2023-06-15 at 12 05 27

- Delete all animals born after Jan 1st, 2022.
-Create a savepoint for the transaction
-Update all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction
Screenshot 2023-06-15 at 12 04 43
Screenshot 2023-06-15 at 12 05 20

- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
svaes
Screenshot 2023-06-15 at 12 08 37

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+-- Transactions begin
+BEGIN;
+UPDATE animals
+SET species='unspecified';
+
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql]: - [ ] according to the requirements you should check if the changes were made before and after rollback.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+
+BEGIN;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species='pokemon'
+WHERE species='';
```
Comment (mikethreels) [inline queries.sql:37]: - [ ] According to the requirements you should commit the changes
- [ ] according to the requirements you should check if the changes were made before and after commit as well.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+
+BEGIN;
+DELETE FROM animals;
+
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql]: - [x] according to the requirements you should check if the changes were made before and after rollback.

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+
+SAVEPOINT my_savepoint;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO my_savepoint
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
```
Comment (mikethreels) [inline queries.sql]: - [ ] According to the requirements you should commit the changes

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Luzinda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job altering the table.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @LDouglasOT , 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- query and update the animal's table perfectly ✔️
- Add a Screenshot of all queries to the PR ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**